# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/scene_building.rs
# Commit: 79f4180c783b
# Full Hash: 79f4180c783b1e72fccb1e49fb8db086ea12ecca
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-18 09:47:25
# Regressor Bug: 1749625
# File Overlap Count: 1
# Description:
#   Bug 1749625 - Fix up and re-enable backdrop-filter r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146643
# ==============================================================================

diff -r 622b05c84338 -r 79f4180c783b gfx/wr/webrender/src/scene_building.rs
--- a/gfx/wr/webrender/src/scene_building.rs	Wed May 18 03:02:04 2022 +0000
+++ b/gfx/wr/webrender/src/scene_building.rs	Wed May 18 03:29:10 2022 +0000
@@ -1696,10 +1696,9 @@
                     &clips,
                 );
             },
-            DisplayItem::BackdropFilter(ref _info) => {
+            DisplayItem::BackdropFilter(ref info) => {
                 profile_scope!("backdrop");
 
-                /*
                 let (layout, _, spatial_node_index, clip_chain_id) = self.process_common_properties(
                     &info.common,
                     None,
@@ -1717,7 +1716,6 @@
                     filter_datas,
                     filter_primitives,
                 );
-                */
             }
 
             // Do nothing; these are dummy items for the display list parser